### DISCUSSION TITLE
How do I set the rocketmq delay message storage time

### DISCUSSION THREAD

**Discussion by Hybdream95** [apache/rocketmq#6826] [Q&A]
现象：
   1.我在测试rocketmq延迟消息，把rocketmq的延迟等级关闭。
    2.之后发送延迟消息时，只要超过了3天时间，rocketmq就会报错。`rocketmqhe delay time should not be bigger than the max delay 259200000ms; `
   3.初步怀疑是消息存储时长过短的原因导致报错。

**Comment by cserwen** [comment]
延时消息超过存储时间之后会被清理的，所以这个行为是禁止的。

**Reply by Hybdream95** [reply]
那我可以修改消息存储时间吗？我看5.0文档中，功能特性里面的-消息存储和清理机制 -最下面： 消息存储时长建议适当增加 。但是没有给出怎么设置消息存储时长。是不可以进行设置吗？

**Reply by Hybdream95** [reply]
如果可以设置的话，怎么进行设置？

**Reply by cserwen** [reply]
应该还是  fileReservedTime=72，单位是小时
